### PR TITLE
Website update 1.5.0 release

### PR DESCRIPTION
**Pull request by sembler**
@tipsy please review these website changes.

### CODE REVIEW

File: website/src/main/java/app/Main.java [hunk 1]
```diff
@@ -7,9 +7,8 @@
 public class Main {
 
     public static void main(String[] args) {
-
         Javalin app = Javalin.create(config ->
-            config.addStaticFiles("src/main/resources/public", Location.EXTERNAL)
+            config.addStaticFiles("website/src/main/resources/public", Location.EXTERNAL)
```
Comment (tipsy) [inline website/src/main/java/app/Main.java:11]: Are you running this from the project root?

**Review by tipsy — APPROVED** [review]
Looks good @sembler !

File: website/src/main/java/app/Main.java (hunk 1 above)
Comment (sembler) [reply website/src/main/java/app/Main.java:11]: My IDE probably was.  It didn't seem to like that this was now a subproject.

Comment (tipsy) [reply website/src/main/java/app/Main.java:11]: Okay, it doesn't matter much, whatever works for you 👍

**Pull request merged**